logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -27,47 +27,63 @@
 # /appengine-vmruntime/vmruntime/cloud_logging.py
 
 import logging.handlers
-import os
 
-from google.cloud.logging.handlers._helpers import format_stackdriver_json
+from google.cloud.logging.handlers.handlers import CloudLoggingHandler
+from google.cloud.logging.handlers.transports import BackgroundThreadTransport
+from google.cloud.logging.resource import Resource
 
-_LOG_PATH_TEMPLATE = '/var/log/app_engine/app.{pid}.json'
-_MAX_LOG_BYTES = 128 * 1024 * 1024
-_LOG_FILE_COUNT = 3
+DEFAULT_LOGGER_NAME = 'python'
 
+EXCLUDED_LOGGER_DEFAULTS = ('google.cloud', 'oauth2client')
 
-class AppEngineHandler(logging.handlers.RotatingFileHandler):
-    """A handler that writes to the App Engine fluentd Stackdriver log file.
+_GLOBAL_RESOURCE = Resource(type='global', labels={})
```

Comment:
Not sure how many places we need this, but if we're bringing it into multiple files might as well just define it once in `resource.py`.